community/modules/scheduler/schedmd-slurm-gcp-v6-controller/modules/slurm_files/scripts/setup.py

```diff
@@ -380,8 +380,8 @@ def configure_dirs():
             sl.unlink()
         sl.symlink_to(tgt)
 
-    for f in ("sort_nodes.py",): # copy auxiliary scripts
-        dst = Path(lookup().cfg.slurm_bin_dir) / f
+    for f in (Path("sort_nodes.py"), Path("tools/task-prolog"), Path("tools/task-epilog")): # copy auxiliary scripts
```

Comment:
I don't understand why `task-prolog` needs to be places in `/usr/local/bin` instead of `/slurm/custom_scripts/`.
Otherwise looks good.

Reply:
Fixed.